strr-strata-web/app/pages/application.vue

```diff
@@ -112,9 +113,12 @@ const handleStrataSubmit = async () => {
 
     // if all steps valid, submit form with store function
     if (isApplicationValid) {
-      const response = await submitStrataApplication()
-      if (response) {
-        return navigateTo(localePath(`/strata-hotel/dashboard/${response.header.applicationNumber}`))
+      const { paymentToken, filingId, applicationStatus } = await submitStrataApplication()
+      const redirectPath = `/strata-hotel/dashboard/${filingId}`
+      if (applicationStatus === ApplicationStatus.PAID) {
+        await navigateTo(localePath(redirectPath))
+      } else {
+        handlePaymentRedirect(paymentToken, redirectPath)
```

Comment:
Should we verify 'applicationStatus' equals 'PAYMENT_DUE' here in case there was some error keeping it in DRAFT or future updates to the api potentially put it straight to one of the review states (maybe the auto approval job runs right as this was added or something like that)

Reply:
Or just if status !== payment due, go to dashboard/id, else go to cc payment ?

Reply:
yeah what you implemented is what I was thinking